docs/current/cli/979595-reference.md

```diff
@@ -113,6 +173,128 @@ Log out of Dagger Cloud:
+```
+
+### Options
+
+| Option               | Description                           |
+| ---------------------| --------------------------------------|
+| `--focus`            | Only show output for focused commands |
+| `-m`, `--mod string` | Path to `dagger.json` config file for the module or a directory containing that file. May be a local path or a remote Git repository |
+
+### Examples
+
+Print the configuration of a local Dagger module:
+
+```shell
+dagger mod -m /path/to/some/dir
+```
+
+Print the configuration of a remote Dagger module:
+
+```shell
+dagger mod -m github.com/dagger/hello-dagger
+```
+
+### Sub-commands
+
+| Sub-command  | Description                                                           |
+| ------------ | --------------------------------------------------------------------- |
+| `init`       | Initialize a new Dagger module in a local directory                   |
+| `install`    | Add a new dependency to a Dagger module                              |
+| `sync`       | Synchronize a Dagger module with the latest version of its extensions |
+| `publish`    | Publish a Dagger module to the Daggerverse                            |
+
+#### dagger mod init
+
+Initialize a new Dagger module in the current directory.
+
+##### Usage
+
+```shell
+dagger mod init [--license string] [--name string] [--root string] [--sdk string]
+```
+
+##### Options
+
+| Option               | Description                                                     |
+| ---------------------| ----------------------------------------------------------------|
+| `--license string`   | License identifier to generate                                  |
+| `--name string`      | Name of the new module                                          |
+| `--root string`      | Root directory that should be loaded for the full module context. Defaults to the parent directory containing `dagger.json` |
+| `--sdk string`       | DK name or image ref to use for the module                      |
+
+##### Example
+
+Initialize a new module named `hello` with the Python SDK:
+
+```shell
+dagger mod init --name hello --sdk=python
+```
+
+#### dagger mod install
+
+Add a new dependency to a Dagger module.
+
+##### Usage
+
+```shell
+dagger mod install
+```
+
+##### Example
+
+Install the `ttlsh` module from the Daggerverse:
+
+```shell
+dagger mod install github.com/shykes/daggerverse/ttlsh@16e40ec244966e55e36a13cb6e1ff8023e1e1473
+```
+
+#### dagger mod sync
+
+Synchronize a Dagger module after a change in its interfaces.
+
+##### Usage
+
+```shell
+dagger mod sync
+```
+
+##### Example
+
+Synchronize a Dagger module after a change in its interfaces.
+
+```shell
+dagger mod sync
+```
+
+#### dagger mod publish
+
+Publish a Dagger module to the Daggerverse.
+
+##### Usage
+
+```shell
+dagger mod publish [--force]
+```
+
+##### Example
+
+Publish the current Dagger module to the Daggerverse.
+
+```shell
+dagger mod publish
+```
```

Comment:
Same here. Example feels unnecessary, but should document the `--force` option.

Reply:
Documented, removed example